lib/optimizer/src/Transformer/AmpRuntimeCss.php

```diff
@@ -170,7 +170,7 @@ private function inlineCss(DOMElement $ampRuntimeStyle, $version)
             $response   = $this->remoteRequest->get($v0CssUrl);
             $statusCode = $response->getStatusCode();
 
-            if (200 < $statusCode || $statusCode >= 300) {
+            if ($statusCode < 200 || $statusCode >= 300) {
```

Comment:
cf https://github.com/ampproject/amp-wp/pull/4686#discussion_r424647168

Reply:
Yes again, a good candidate for backporting.

Reply:
Note, the changes are all in separate commits, so backporting the above should be a straight-forward cherry-pick.

Reply:
The branch doesn't merge cleanly into 1.5, and 83018f2168f2fa48ac21724f11ac01ea94d85855 doesn't merge cleanly either. But I'll apply this one fix manually, and leave the other changes to 1.6 alone.

Reply:
See f9f10d121edc059ab55530d0e0ccf21d25ffd65b.